internals/cli/cmd_run.go

```diff
@@ -60,8 +60,8 @@ var sharedRunEnterArgsHelp = map[string]string{
 }
 
 type cmdRun struct {
-	clientMixin
 	sharedRunEnterOpts
+	client *client.Client
```

Comment:
Space precedes client elsewhere.

Reply:
But here we don't have more than one struct embedding so I don't think it makes sense to separate the members with a space.

Reply:
For the record, Fred is on a good line of thinking in his reviews: he's looking for and suggesting organization and consistency. Might be worth paying a bit more attention to his nitpicks.